### PR TITLE
Observation Groupes Homogenes de malades

### PR DESCRIPTION
**Pull request by lnmylos**
## Fixes

Fixes #[issue number] by @[issue author]

## Description
Resource Observation for Groupes Homogènes de Malades

### CODE REVIEW

File: input/fsh/SD_PMSIGHM.fsh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Profile:        PMSIGHM
+Parent:         Observation
+Id:             pmsi-grouped-homogene-maladesd-dpdr
+Title:          "Groupes homogènes de Malades"
+Description:    "Profil pour la ressource Groupes Homogènes de Malades du PMSI"
```
Comment (nriss) [inline input/fsh/SD_PMSIGHM.fsh:5]: I don't think you have to write "Profil pour la ressource", it doesn't give any information
Maybe something like "Observation sur le Groupe Homogène de Malade (GHM) d'un patient"

File: input/fsh/SD_PMSIGHM.fsh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* identifier MS
+//value= N° de Groupage
+
+
+* category MS
+// Code majeure de diagnostic: https://www.atih.sante.fr/manuel-des-ghm-version-11d
```
Comment (nriss) [inline input/fsh/SD_PMSIGHM.fsh:12]: majeur

**Review by nriss — CHANGES_REQUESTED** [review]

File: input/fsh/SD_PMSIGHM.fsh (hunk 2 above)
Comment (lnmylos) [reply input/fsh/SD_PMSIGHM.fsh:12]: done!

File: input/fsh/SD_PMSIGHM.fsh (hunk 1 above)
Comment (lnmylos) [reply input/fsh/SD_PMSIGHM.fsh:5]: 👍

File: input/fsh/SD_PMSIGHM.fsh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+Profile:        PMSIGHM
+Parent:         Observation
+Id:             pmsi-grouped-homogene-maladesd-dpdr
```
[the remark this answers is not in the document]
Comment (lnmylos) [reply input/fsh/SD_PMSIGHM.fsh:3]: 👍

**Comment by lnmylos** [comment]
GHM and Encounter RSS ready.

**Review by nriss — APPROVED** [review]
🌱  ➡️  🌴

**Pull request merged**